src/patternfly/components/Tabs/tabs.scss

```diff
@@ -233,8 +286,12 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
     // Add border right color and weight
     .pf-c-tabs__item.pf-m-current {
       --pf-c-tabs__link--before--BorderRightColor: var(--pf-c-tabs__item--m-current__link--Background);
-      --pf-c-tabs__link--before--BorderBottomColor: var(--pf-c-tabs__link--before--BorderColor);
+      --pf-c-tabs__link--before--BorderBottomColor: var(--pf-c-tabs__link--before--border-color--base);
       --pf-c-tabs__link--before--BorderBottomWidth: var(--pf-c-tabs__link--before--BorderWidth);
+
+      &:first-child {
+        --pf-c-tabs__link--before--BorderTopWidth: var(--pf-c-tabs__link--before--BorderWidth);
```

Comment:
should `--pf-c-tabs__link--before--BorderWidth` be `--pf-c-tabs__link--before--border-width--base`?